CrawlerTests/ScienceDailyScraperTests.cs

```diff
@@ -13,15 +14,25 @@ namespace CrawlerTests
     public class ScienceDailyScraperTests
```

Comment:
Is this test class test ScienceDaily specifically?

Reply:
Fixed